src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Beep.cs

```diff
@@ -8,7 +8,7 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32, SetLastError = true)]
-        internal static extern bool Beep(int frequency, int duration);
+        [GeneratedDllImport(Libraries.Kernel32, SetLastError = true)]
+        internal static partial bool Beep(int frequency, int duration);
```

Comment:
We should consider these slightly differently. If we change the `bool` here to an `int` it would be blittable and therefore could be inlined. Is there some way to we could imbue the analyzer/fixer to issue a warning that this could be made better with minimal changes?

Reply:
Since the analyzer flags the non-inline-able p/invokes and we're basically going to end up looking through them all individually for the runtime repo, it is worth thinking about as we see them.

I think having an analyzer/fixer do something gets pretty annoying in terms of a metric for 'minimal'. If this is called by a bunch of places that used the returned `bool`, all those need to change - or a wrapper added that converts the returned `int` to a `bool` (essentially what the source generator would do).

This particular one also has `SetLastError`.